pkgs/applications/networking/mailreaders/thunderbird/sequoia.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, thunderbird, sequoia-octopus-librnp }:
+let
+  originalPackage = thunderbird;
+
+  # We use `overrideAttrs` instead of defining a new `mkDerivation` to keep
+  # the original package's `output`, `passthru`, and so on.
+  thunderbird-sequoia = originalPackage.overrideAttrs (old: rec {
```

Comment:
```suggestion
# We use `overrideAttrs` instead of defining a new `mkDerivation` to keep
# the original package's `output`, `passthru`, and so on.
thunderbird.overrideAttrs (old: rec {
```